backend/notifications/models.py

```diff
@@ -0,0 +1,184 @@
+from django.db import transaction
+from django.db.models import Q, UniqueConstraint
+from django.utils.safestring import mark_safe
+from django.db import models
+from users.models import User
+from notifications.template_utils import render_template_from_string
+from notifications.querysets import EmailTemplateQuerySet, SentEmailQuerySet
+from model_utils.models import TimeStampedModel
+from django.utils.translation import gettext_lazy as _
+from django.template.loader import render_to_string
+
+
+class EmailTemplate(TimeStampedModel):
```

Comment:
add the concept of system_template for templates like reset password?

main use case: reset password but how would it work in a passwordless world? technically would only be needed by pycon.it unless we move to serverless there